rust/src/push/base_rules.rs

```diff
@@ -596,6 +610,68 @@ pub const BASE_APPEND_UNDERRIDE_RULES: &[PushRule] = &[
         default: true,
         default_enabled: true,
     },
+    PushRule {
+        rule_id: Cow::Borrowed("global/underride/.org.matrix.msc3930.rule.poll_start_one_to_one"),
+        priority_class: 1,
+        conditions: Cow::Borrowed(&[
+            Condition::Known(KnownCondition::RoomMemberCount {
+                is: Some(Cow::Borrowed("2")),
+            }),
+            Condition::Known(KnownCondition::EventMatch(EventMatchCondition {
+                key: Cow::Borrowed("type"),
+                pattern: Some(Cow::Borrowed("org.matrix.msc3381.poll.start")),
```

Comment:
I think this is fine, but the MSC doesn't really mention matching the unstable events. 🤷

Reply:
Oh sorry, I should have mentioned the discussion I had on the MSC either somewhere in this PR or/in addition to in a comment in the code: https://github.com/matrix-org/matrix-spec-proposals/pull/3930#discussion_r1062687762

Though I'll note that the conclusion of that discussion has not influenced the MSC text itself yet.

Reply:
Ah, that's fair. I'll see if I can get a line added that says something to the effect of "While MSC3381 has not been accepted, `org.matrix.msc3381.poll.*` may be used in place of `m.poll.*`".

Reply:
I'm concerned about the text in MSC3930 *not* MSC3381. I could very easily see someone (i.e. a client) implementing the "unstable version" with different values.

Reply:
Oof, I meant to comment on MSC3930. too many mscs...

Reply:
From the conclusion of https://github.com/matrix-org/matrix-spec-proposals/pull/3930#discussion_r1067344741 (which has yet to be put into the spec text, but :shrug:), it's up to the implementation what exact prefix is used for the push rules. That's a very spec-way of saying that in our case, we can use `org.matrix.msc3381.poll.*`.